events/views.py

```diff
@@ -90,3 +93,20 @@ def deleteEvent(request, event_id):
 def eventDetail(request, event_id):
     event = get_object_or_404(Event, pk=event_id)
     return render(request, "events/event-detail.html", {"event": event})
+
+
+# Map Code
+
+
+def get_data(request):
+    location_data = Event.objects.all()
```

Comment:
can we add filter here for is_active

Reply:
Okay will do

Reply:
Change has been made. Please take a look